client/js/components/supporter-fields.js

```diff
@@ -52,11 +52,12 @@ function view(state) {
       h('input.u-marginBottom--0', {
         props: {
           type: 'email'
-        , title: emailTitle
-        , name: 'email'
-        , required: state.required.email
-        , value: state.supporter.email
-        , placeholder: emailTitle
+          , title: emailTitle
+          , name: 'email'
+          , pattern: "^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\\.[a-zA-Z0-9-]+)+$"
```

Comment:
What was the pattern we originally had?

My concern about this pattern is that it will fail for any username or domain with non-ASCII characters. That seems like a recipe for complaints :)

Reply:
The original format that `type=email` uses is:

`/^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/`

So it ended just being the last `*` replaced for `+`.

Reply:
Ah. Where is that documented?

Reply:
So, I looked tested this in Firefox and Chrome. The default email format does not allow non-ASCII usernames. I guess we can forward with this and pray that we don't have a problem in the future. *ugh*